package.json

```diff
@@ -29,7 +29,7 @@
     "sls:invoke:rrp-beacon-update": "sls invoke --function rrp-beacon-update",
     "sls:invoke-local:rrp-beacon-update": "yarn build && sls invoke local --function rrp-beacon-update",
     "sls:invoke:psp-beacon-update": "sls invoke --function psp-beacon-update",
-    "sls:invoke-local:psp-beacon-update": "yarn build && sls invoke local --function psp-beacon-update",
+    "sls:invoke-local:psp-beacon-update": "yarn build && sls invoke local --function psp-beacon-update --param='provider=local'",
```

Comment:
Oh, I guess this is the `param:provider` I was asking about :smile: Yeah, that makes sense :slightly_smiling_face: